### PR TITLE
Add contract proto

### PR DESCRIPTION
**Pull request by AgwaB**
details:
Add contract proto

`Contract` contains information about the function that the user called.
I think we should talk more with the compiler... @hihiboss  @junbeomlee

### CODE REVIEW

File: vm/contract.go [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package vm
+
+type Contract struct {
```
Comment (hihiboss) [inline vm/contract.go:19]: If there are comments for `Code` and `Input`, it would be better.

Comment (AgwaB) [reply vm/contract.go:19]: I added that!

File: vm/vm_test.go [hunk 2]
```diff
@@ -17,9 +17,10 @@
 package vm
 
 import (
-	"reflect"
 	"testing"
 
+	"reflect"
+
 	"github.com/DE-labtory/koa/opcode"
 	"github.com/stretchr/testify/assert"
```
Comment (joomanzi) [inline vm/vm_test.go]: still we use this lib?

Comment (AgwaB) [reply vm/vm_test.go]: Maybe i made a mistake... :( 
thx!!

**Pull request closed without merging**